src/include/duckdb/main/settings.hpp

```diff
@@ -596,7 +596,8 @@ struct StreamingBufferSize {
 struct MaximumTempDirectorySize {
 	static constexpr const char *Name = "max_temp_directory_size";
 	static constexpr const char *Description =
-	    "The maximum amount of data stored inside the 'temp_directory' (when set) (e.g. 1GB)";
+	    "If the `temp_directory` is set to an existing directory, this option defaults to the available disk space on "
```

Comment:
This now only describes the default value - should we leave in the description of the setting?

Reply:
Yes, I pushed an update with this